packages/theming/atlas/scripts/build.js

```diff
@@ -1,65 +1,82 @@
-const { rm, cd } = require("shelljs");
-const { join } = require("path");
 const concurrently = require("concurrently");
+const { access } = require("fs").promises;
+const { join } = require("path");
+const { rm } = require("shelljs");
 
-const mode = process.argv[2] || "build";
-const MX_PROJECT_PATH = process.env.ATLAS_MX_PROJECT_PATH; // should be an absolute path.
-let outputProjectDir = MX_PROJECT_PATH ? MX_PROJECT_PATH : join(__dirname, "../tests/testProject");
-let outputThemeDir;
-let outputThemeSourceDir;
-let projectDeployDirWeb;
+main().catch(e => {
+    console.error(e);
+    process.exit(-1);
+});
 
-switch (mode) {
-    case "build":
-    case "start":
-        outputThemeDir = join(outputProjectDir, "theme");
-        outputThemeSourceDir = join(outputProjectDir, "themesource/atlas_ui_resources");
-        projectDeployDirWeb = join(outputProjectDir, "deployment/web");
-        break;
-    case "release":
-        outputThemeDir = join(__dirname, "../dist/theme");
-        outputThemeSourceDir = join(__dirname, "../dist/themesource");
-        break;
-}
+async function main() {
+    const mode = process.argv[2] || "build";
+
+    let outputDir;
+
+    if (mode === "build" || mode === "start") {
+        const MX_PROJECT_PATH = process.env.ATLAS_MX_PROJECT_PATH; // should be an absolute path.
+        outputDir = MX_PROJECT_PATH ? MX_PROJECT_PATH : join(__dirname, "../tests/testProject");
+
+        const toRemoveDirs = [join(outputDir, "theme"), join(outputDir, "themesource/atlas_ui_resources")];
+        const toRemoveDirsExist = await Promise.all(toRemoveDirs.map(async dir => exists(dir)));
+
+        if (toRemoveDirsExist.includes(true)) {
```

Comment:
Why do we need to check? `rm` will not fail, and also the first or second argument can also not exist and the `rm` will still be executed. 

Suggestion: Just remove the if..

Reply:
I know. I did this for the messages. The messages ensure the user of the script that the folders have been cleaned out, so that they know files not being overwritten during a copy won't exist anymore. Therefore, I would like to keep this.